app/components/UI/FiatOnRampAggregator/components/ScreenRegion.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import { StyleSheet, View, SafeAreaView, ScrollView, ViewStyle } from 'react-native';
+import { colors } from '../../../../styles/common';
+import TextJS from '../../../Base/Text';
+
+const Text = TextJS as any;
+
+interface Style {
+	wrapper: ViewStyle;
+	container: ViewStyle;
+	content: ViewStyle;
+	header: ViewStyle;
+	body: ViewStyle;
+	footer: ViewStyle;
+}
+
+export const styles = StyleSheet.create<Style>({
+	wrapper: {
+		flex: 1,
+	},
+	container: {
+		backgroundColor: colors.white,
+		flex: 1,
+	},
+	content: {
+		padding: 15,
+	},
+	header: {
+		marginTop: 22,
+		marginBottom: 22,
+		alignItems: 'center',
```

Comment:
According to the design the header margin is smaller (16)

```suggestion
	header: {
		marginVertical: 16,
		alignItems: 'center',
```